cmd/fsi.go

```diff
@@ -0,0 +1,88 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/qri-io/ioes"
+	"github.com/qri-io/qri/lib"
+	"github.com/spf13/cobra"
+)
+
+// NewFSICommand creates a new `qri fsi` command for working with file system
+// integration
+func NewFSICommand(f Factory, ioStreams ioes.IOStreams) *cobra.Command {
+	o := &FSIOptions{IOStreams: ioStreams}
+	cmd := &cobra.Command{
+		Use:    "fsi",
+		Hidden: true,
+		Short:  "file system integration tools",
+	}
+
+	link := &cobra.Command{
+		Use:   "link",
+		Short: "link a .qri-ref",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			if err := o.Complete(f, args); err != nil {
+				return err
+			}
+			return o.Link()
+		},
+	}
+
+	unlink := &cobra.Command{
+		Use:   "unlink",
+		Short: "unlink a .qri-ref",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			if err := o.Complete(f, args); err != nil {
+				return err
+			}
+			return o.Unlink()
+		},
+	}
+
```

Comment:
In the future, would also be nice to have "cleanup", which goes through the .fbs and removes any folders that it can't find anymore. Also "relink", which takes a directory and dsref and moves which directory the .fbs points to for that dsref, removing the old directory's .qri-ref. Or perhaps "link" could just do that.

Reply:
totally. I'll file an issue.